plugins/etc/httpd/conf.d/pulp_rpm.conf

```diff
@@ -16,19 +16,23 @@
 AddType application/x-pkcs7-crl .crl
 AddType application/x-x509-ca-cert .crt
 
-# -- HTTPS Repositories ---------
-Alias /pulp/repos /var/www/pub/yum/https/repos
+# -- Yum Repositories ---------
+<Location /pulp/repos/>
+  RewriteEngine On
+  RewriteCond %{HTTPS} on
+  RewriteRule (.+/pulp/repos/)(.+) /pulp/content/var/www/pub/yum/https/repos/$2 [DPI]
+  RewriteCond %{HTTPS} off
+  RewriteRule (.+/pulp/repos/)(.+) /pulp/content/var/www/pub/yum/http/repos/$2 [DPI]
+</Location>
 
 # -- HTTPS Exports
 Alias /pulp/exports /var/www/pub/yum/https/exports
 
 <Directory /var/www/pub/yum/https>
-
     WSGIAccessScript /srv/pulp/repo_auth.wsgi
-
     SSLRequireSSL
-    SSLVerifyClient optional_no_ca
-    SSLVerifyDepth 2
+    SSLVerifyClient optional
```

Comment:
Whoah! We really need to audit our configs ☹

https://pulp.plan.io/issues/1257